concept/type/impl/ThingTypeImpl.java

```diff
@@ -373,21 +393,18 @@ public void unsetPlays(RoleType roleType) {
     }
 
     @Override
-    public FunctionalIterator<RoleTypeImpl> getPlays() {
-        if (isRoot()) return Iterators.empty();
-        Set<TypeVertex> overridden = new HashSet<>();
-        vertex.outs().edge(Encoding.Edge.Type.PLAYS).overridden().filter(Objects::nonNull).forEachRemaining(overridden::add);
+    public Seekable<RoleType, Order.Asc> getPlays() {
+        if (isRoot()) return emptySorted();
         assert getSupertype() != null;
-        return link(
-                vertex.outs().edge(Encoding.Edge.Type.PLAYS).to().map(v -> RoleTypeImpl.of(graphMgr, v)),
-                getSupertype().getPlays().filter(att -> !overridden.contains(att.vertex))
-        );
```

Comment:
TODO this behaviour is not exactly replicated in the new change! We should only filter the supertypes' plays as we do in the old code